internal/outputs/scc/scc.go

```diff
@@ -309,8 +272,8 @@ func mapFindingSourceProperties(finding *Finding) map[string]*structpb.Value {
 	result["PolicyName"] = structpb.NewStringValue(finding.SourcePolicyName)
 	result["PolicyFile"] = structpb.NewStringValue(finding.SourcePolicyFile)
 	result["PolicyGroup"] = structpb.NewStringValue(finding.SourcePolicyGroup)
-
 	result["Recommendation"] = structpb.NewStringValue(finding.Recommendation)
+	result["GKEPolicyAutomationVersion"] = structpb.NewStringValue(version.Version)
```

Comment:
i am wondering if we shouldn't keep policy version instead of tool version - we can run the tool on any given policy set outside the tool itself

Reply:
That would be good idea but we don't have policy versioning for now (other than git revisions).
We can add policy version as additional attribute in future - it is not contradicting with tool's version in my opinion.

The rationale behind adding this was to give possibility to track which version of a tool created (or updated) given finding in SCC.

Example of source properties in SCC:

![image](https://user-images.githubusercontent.com/20063540/202552842-ad6329b2-d425-4b34-82af-4aaaa2cfb901.png)